sql/core/src/test/resources/sql-tests/inputs/order-by-all.sql

```diff
@@ -0,0 +1,44 @@
+create temporary view data as select * from values
```

Comment:
nit: we can have a test case combining `group by all` and `order by all`. It should work, just for covering a testing scenario.

Reply:
isn't it to better to test different features in test golden files?

Reply:
Oh you mean test both feature in one query? That's a good idea, we can add later.